### PR TITLE
add information about step limit for pipeline upload file

### PR DESCRIPTION
**Pull request by nsuma8989**

### CODE REVIEW

**Comment by nsuma8989** [comment]
@mbelton-buildkite Here is the PR based on comment https://github.com/buildkite/docs/pull/1922#pullrequestreview-1338479689

File: clicommand/pipeline_upload.go [hunk 1]
```diff
@@ -43,7 +43,9 @@ Description:
    - buildkite/pipeline.json
 
    You can also pipe build pipelines to the command allowing you to create
-   scripts that generate dynamic pipelines.
+   scripts that generate dynamic pipelines.The configuration file has a
```
Comment (mbelton-buildkite) [inline clicommand/pipeline_upload.go:46]: Missing a space. You could also consider starting a new paragraph for this.

```suggestion
   scripts that generate dynamic pipelines. The configuration file has a
```

**Review by mbelton-buildkite — APPROVED** [review]
One small fix to make but otherwise looks good to me. I assume you'll get someone from Dispatch to also review this change (maybe @moskyb?)

**Review by triarius — APPROVED** [review]
LGTM

**Comment by nsuma8989** [comment]
@mbelton-buildkite Please let me know if we are good with this change ?

**Comment by mbelton-buildkite** [comment]
Looks good to me :)

**Comment by nsuma8989** [comment]
@triarius Please can you help with merging this PR as I dont see the build as passed and do not have permissions to merge

**Comment by triarius** [comment]
@nsuma8989 I've kicked off a build and enabled automerge. It should merge automatically when green.

**Pull request merged**